src/main/java/org/jabref/gui/fieldeditors/FieldNameLabel.java

```diff
@@ -14,5 +18,39 @@ public FieldNameLabel(Field field) {
         setPadding(new Insets(4, 0, 0, 0));
         setAlignment(Pos.CENTER);
         setPrefHeight(Double.POSITIVE_INFINITY);
+        setTip(field);
+    }
+
+    public void setTip(Field field) {
+        Tooltip tip = new Tooltip();
+        if (field.isStandardField()) {
+            StandardField standardField = (StandardField) field;
+            switch (standardField) {
+                case AUTHOR:
+                    tip.setText(Localization.lang("The name(s) of the author(s), in the format described in the LaTeX book. Remember, all names are separated with the and keyword, and not commas."));
+                    break;
+                case JOURNAL:
+                    tip.setText(Localization.lang("Journal name. Abbrevations may be used."));
+                    break;
+                case TITLE:
+                    tip.setText(Localization.lang("The work's title"));
+                    break;
+                case YEAR:
+                    tip.setText(Localization.lang("The year of publication or, for an unpublished work, the year it was written. Generally it should consist of four numerals, such as 1984, although the standard styles can handle any year whose last four nonpunctuation characters are numerals, such as '(about 1984)'."));
+                    break;
+                default:
+                    return;
+            }
+        } else if (field instanceof InternalField) {
+            InternalField internalField = (InternalField) field;
+            switch (internalField) {
+                case KEY_FIELD:
+                    tip.setText(Localization.lang("[First author'last name][Article year], e.g. Jones2020"));
+                    break;
+                default:
+                    return;
+            }
+        }
+        this.setTooltip(tip);
```

Comment:
Currently, tooltips are only added for five fields (`author`, `journal`, `title`, `year` and `key`). It would be really great, if the other fields (e.g. in the other tabs of the entry editor) would get tooltips as well. Examples: `date`, `editor`, `DOI`, `pages`, `number`, `language`, `volume`, `urldate`, ...

Reply:
Yes, what I made is just a start.

Reply:
This is really good to hear! Thank you for your effort! :)